integration-tests/src/test/java/tachyon/client/keyvalue/KeyValuePartitionIntegrationTest.java

```diff
@@ -76,4 +78,52 @@ public void readerWriterTest() throws IOException, TachyonException {
     Assert.assertArrayEquals(VALUE2, mKeyValuePartitionReader.get(KEY2));
     Assert.assertNull(mKeyValuePartitionReader.get("NoSuchKey".getBytes()));
   }
+
+  @Test
+  public void sizeTest() throws Exception {
+    byte[][] keys = new byte[][]{KEY1, KEY2};
```

Comment:
The number of elements is deterministic, what's the pro of a `List<byte[]>` instead of `byte[][]`?